changelog.d/2366.changed.md

```diff
@@ -0,0 +1,3 @@
+Changed port stealing configuration:
+1. Added new `ports` field to the `incoming` configuration. The field lists ports that should be stolen/mirrored. Other ports remain local.
```

Comment:
Logic should be that if `ignore_ports` exists and `ports` is empty, then all ports will be stolen except for those in `ignore_ports`

Reply:
The new field itself is optional, so I think the logic checks out. Should I extend the changelog here?

Reply:
nvm I think I was confused